pydbtools/utils.py

```diff
@@ -122,8 +123,34 @@ def get_database_name_from_userid(user_id: str) -> str:
     return unique_db_name
 
 
+def get_database_name_from_sql(sql: str) -> str:
+    """
+    Obtains database name from SQL query for use
+    by awswrangler.
+
+    Args:
+        sql (str): The raw SQL query as a string
+
+    Returns:
+        str: The database table name
+    """
+
+    for table in sql_metadata.Parser(sql).tables:
+        # Return the first database seen in the
+        # form "database.table"
+        xs = table.split(".")
+        if len(xs) > 1:
+            return xs[0]
+
+    # Return default in case of failure to parse
+    return None
+    # Fails tests but might give fewer errors...
+    # return "unknown_database"
```

Comment:
remove this comment

Reply:
https://github.com/moj-analytical-services/pydbtools/pull/57/commits/2cc5bb444212ad8f42bb68fddf8ef023ceb4d1b8